File: php/examples/get_registered_agent_products.php
<?php
require_once 'base_request.php';

// Example of GET /registered-agent-products

$debug = true;
$website_url = $_ENV['WEBSITE_URL'];
$request_data = '';
$request_params = [
	'url' => $website_url
];

send_request('GET', '/registered-agent-products', $request_params, $request_data);

File: csharp/examples/GetRegisteredAgentProducts.cs
using System;
namespace Examples.examples
{
    // Example of GET /registered-agent-products

    public class GetRegisteredAgentProducts : BaseRequest
    {
        private string _websiteUrl;

		public GetRegisteredAgentProducts()
		{
            _websiteUrl = Environment.GetEnvironmentVariable("WEBSITE_URL");
            Console.WriteLine($"GetRegisteredAgentProducts: _websiteUrl={_websiteUrl}");
        }

        public override string SendRequest()
        {
            return GetRequest($"registered-agent-products?url={_websiteUrl}");
        }
    }
}

